templates/includes/sidebar.html

```diff
@@ -1,6 +1,12 @@
 <ul class="sidebar-nav">
-  <li><a href="{% url "taxi:index" %}">Home page</a></li>
-  <li><a href="{% url "taxi:manufacturer-list" %}">Manufacturers</a></li>
-  <li><a href="{% url "taxi:car-list" %}">Cars</a></li>
-  <li><a href="{% url "taxi:driver-list" %}">Drivers</a></li>
+    {% if user.is_authenticated %}
+        <p>User: {{ user.username }}</p>
+        <a href="{% url 'logout' %}">Logout</a>
+    {% else %}
+        <a href="{% url 'login' %}">Click here to login</a>
+    {% endif %}
+    <li><a href="{% url 'taxi:index' %}">Home page</a></li>
+    <li><a href="{% url 'taxi:manufacturer-list' %}">Manufacturers</a></li>
```

Comment:
Ensure that the URL name 'taxi:manufacturer-list' is defined in your `taxi.urls` module. If not, update the URL name to match the actual name used in your URL configurations.